framework/include/tinyalsa/tinyalsa.h

```diff
@@ -331,7 +331,7 @@ unsigned int pcm_bytes_to_frames(const struct pcm *pcm, unsigned int bytes);
  * @param[out] pcm         A PCM handle.
  * @param[in]  data        The audio sample array
  * @param[in]  frame_count The number of frames occupied by the sample array.
- * @return On success,  written number of frames returned. On failure, a negative number returned.
+ * @return On success, written number of frames returned. On failure, a negative number returned.
```

Comment:
you should change permission all files.

Reply:
Sorry, i dont understand your comment. Can you please explain?

Reply:
currently, all files changed file permission 100644->100755

Reply:
Fixed